doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -315,36 +329,36 @@ set with:
 
 .. _bluestore-rocksdb-sharding:
 
-RocksDB Sharding
+rocksdb sharding
 ================
 
-Internally BlueStore uses multiple types of key-value data,
-stored in RocksDB.  Each data type in BlueStore is assigned a
-unique prefix. Until Pacific all key-value data was stored in
-single RocksDB column family: 'default'.  Since Pacific,
-BlueStore can divide this data into multiple RocksDB column
-families. When keys have similar access frequency, modification
-frequency and lifetime, BlueStore benefits from better caching
-and more precise compaction. This improves performance, and also
+internally bluestore uses multiple types of key-value data,
```

Comment:
Why lowercase the heading? and the follow sentences?

BlueStore stores multiple types of key-value data in an internal RocksDB database.

Reply:
See https://github.com/ceph/ceph/pull/51771#issuecomment-1564794725.